### ISSUE TITLE
Export Style.json file contains Tilehosting.com 

### ISSUE THREAD

**Issue by keum** [maputnik/editor#308]
Noticed two days ago that when I export style.json from Maputnik that Tilehosting.com has changed to Maptiler Cloud.com but no error has generated yet. 

Here is the code 
'''
"sources": {
    "openmaptiles": {
      "type": "vector",
      **"url": "https://free.tilehosting.com/data/v3.json?key={key}"**
    }
  },
  "sprite": "https://openmaptiles.github.io/klokantech-basic-gl-style/sprite",
  "glyphs": "https://free.tilehosting.com/fonts/{fontstack}/{range}.pbf?key={key}",
'''
Would be nice to update with maptiler cloud.com, unless it's suppose to be. 

thanks!

**Comment by orangemug** [comment]
@keum when I look at my `cloud.maptiler.com` account I still see `https://maps.tilehosting.com/data/v3.json?key=KEY` as the dataset url, do you see a different URL?

**Comment by keum** [comment]
@orangemug, tilehosting still works, it's just that I got confused with Maptiler.com and Tilehosting.com...took me awhile to figure out they're two same thing..dud. btw workshop went well and people really liked how we can style our own data locally using local maputnik...loved that whole offline option to work. 

I have separate question about styling by data type, wonder if i need to write a ticket to the community and ask for it? want to catagorize symbol by field value. let me know how i should go about asking this of how-to?

**Comment by orangemug** [comment]
Ok so I'm really unclear which of the `tilehosting.com`/`openmaptiles.com`/`cloud.maptiler.com` we should be using, also the name OpenMapTiles in the data sources panel no longer seems correct 😕 

@klokan do you want to comment? I guess we should probably get your new brand/url correct to avoid confusion.

**Comment by orangemug** [comment]
> btw workshop went well and people really liked how we can style our own data locally using local maputnik...loved that whole offline option to work.

Nice, congrats @keum! Please share any feedback about what worked well and what didn't or anything we can improve on in the editor. Also if you have any general statements or photos of the event don't forget about #164 😄

**Comment by klokan** [comment]
> I'm really unclear which of the tilehosting.com/openmaptiles.com/cloud.maptiler.com we should be using, also the name OpenMapTiles in the data sources panel no longer seems correct 😕

There is no change required on Maputnik side now. OpenMapTiles for the source is correct - we are keen on providing the key for free public Maputnik instance - as we did from the day one of this open-source project.

The official hosting for OpenMapTiles tiles is now called MapTiler: https://www.maptiler.com/

Domains for tiles remain to be the same now. Any change in future means we set redirects and announce that properly to everybody who has registered a free account at https://cloud.maptiler.com/
